packages/sdk/src/evm/common/any-evm-utils.ts

```diff
@@ -888,7 +926,7 @@ export async function fetchAndCacheDeployMetadata(
   return data;
 }
 
-function estimateGasForDeploy(initCode: string) {
+export function estimateGasForDeploy(initCode: string) {
```

Comment:
why did this didn't work for the zk chains? do we just need to tweak some of the values in there?